src/priceHistory/saga.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Price,
+  fetchPriceHistoryFailure,
+  fetchPriceHistoryStart,
+  fetchPriceHistorySuccess,
+} from 'src/priceHistory/slice'
+import Logger from 'src/utils/Logger'
+import { ensureError } from 'src/utils/ensureError'
+import { fetchWithTimeout } from 'src/utils/fetchWithTimeout'
+import { safely } from 'src/utils/safely'
+import networkConfig from 'src/web3/networkConfig'
+import { call, put, spawn, takeLatest } from 'typed-redux-saga'
+
+const TAG = 'priceHistory/saga'
+
+/** @beta - Exclude from Knip dep check */
```

Comment:
```suggestion
/** @beta - Exclude from Knip unused code check */
```

Reply:
might be worth noting in the docstring what the defaults are if start/end timestamps are excluded

Reply:
Added in ddff343278ceb7d786ffb4ba5ca36344bedf9019!